specification/components/schemas/contractCertificateBundle.v1.json

```diff
@@ -5,9 +5,9 @@
     "x-examples": {
         "example-1": {
             "contractCertificateChain": {
-                "certificate": "",
-                "subCa2Certificate": "",
-                "subCa1Certificate": "",
+                "certificate": "MIICQDCCAeWgAwIBAgICBNMwCgYIKoZIzj0EAwIwRDELMAkGA1UEBhMCREUxDTALB",
```

Comment:
example is not correct :  
* Missing pcid and emaid.
* contractSignatureEncryptedPrivateKey and dHpublickey needs to be out of contractCertificateChain
* Maybe we can put the base64 of real (test) certificates instead of a truncated one ?

Also this MR is in conflict with https://github.com/charinev/opnc/pull/49 that removed this example.